analytics_data_api/v0/views/problems.py

```diff
@@ -104,7 +112,22 @@ class ProblemResponseAnswerDistributionView(generics.ListAPIView):
     def get_queryset(self):
         """Select all the answer distribution response having to do with this usage of the problem."""
         problem_id = self.kwargs.get('problem_id')
-        return ProblemResponseAnswerDistribution.objects.filter(module_id=problem_id)
+        consolidate = self.request.QUERY_PARAMS.get('consolidate', 'False')
```

Comment:
I also forgot to mention to @dylanrhodes earlier that we'd like this to be called `consolidate_variants`, to be a bit more verbose and clearer.